utils.js

```diff
@@ -132,13 +133,17 @@ module.exports = {
 
   /**
    * Simplify superagent's http response object
-   * @param {object} r
+   * @param {object} r - The response.
+   * @param {object} Request - The superagent's Request instance.
+   * @param {object} props  - The request properties.
    */
-  gutResponse: function(r){
+  gutResponse: function(r, Request, props){
```

Comment:
Could we replace `r` with a more meaningful name? This was very hard to read. Same for the `d` further below in this function